environments/sbox/sbox.tfvars

```diff
@@ -268,4 +268,97 @@ frontends = [
       },
     ]
   },
+  {
+    name             = "idam-web-admin"
+    custom_domain    = "idam-web-admin.sandbox.platform.hmcts.net"
```

Comment:
missing public dns for this / afdverify?

Reply:
https://github.com/hmcts/azure-public-dns/pull/236